src/components/Common/SingleService.js

```diff
@@ -0,0 +1,16 @@
+import React, {Component} from 'react';
+class SingleService extends Component{
```

Comment:
Same here. If component doesn't uses a state, let's convert it into a functional component instead.